packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -93,11 +100,41 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
         <wui-tooltip-select text="Send" icon="send"></wui-tooltip-select>
       </wui-flex>
 
-      <wui-tabs localTabWidth="120px" .tabs=${ConstantsUtil.ACCOUNT_TABS}></wui-tabs>
+      <wui-tabs
+        .onTabChange=${this.onTabChange.bind(this)}
+        .activeTab=${this.currentTab}
+        localTabWidth="104px"
```

Comment:
is this ok ?

Reply:
Yes normally `localTabWidth` is `100px`, but we set `wui-tabs` to `width:100%`, so `100px` is not enough anymore.